aws_securityhub/help.md

```diff
@@ -121,24 +121,22 @@ This trigger is used to poll from a SQS Queue.
 
 |Name|Type|Default|Required|Description|Enum|
 |----|----|-------|--------|-----------|----|
-|AttributeNames|[]string|['All']|False|A list of s that need to be returned along with each message|None|
+|AttributeNames|[]string|['All']|False|A list of attributes that need to be returned along with each message ['All','Policy','VisibilityTimeout','MaximumMessageSize','MessageRetentionPeriod','ApproximateNumberOfMessages','ApproximateNumberOfMessagesNotVisible','CreatedTimestamp','LastModifiedTimestamp','QueueArn','ApproximateNumberOfMessagesDelayed','DelaySeconds','ReceiveMessageWaitTimeSeconds','RedrivePolicy','FifoQueue','ContentBasedDeduplication','KmsMasterKeyId','KmsDataKeyReusePeriodSeconds']|None|
```

Comment:
I'm not sure what you were trying to do here. Is that supposed to be in the enum?

Reply:
It's what the validator wants

Reply:
That's got to be a mistake with the validator. You need a pipe between the description and the enum values.

Reply:
Look at the rich view of the file. The enum values show up in the description column.